content/ref/python/watch.md

```diff
@@ -22,7 +22,12 @@ extended to support arbitrary machine learning models in the future.
 
 | Args |  |
 | :--- | :--- |
-|  models (Union[torch.nn.Module, Sequence[torch.nn.Module]]): A single model or a sequence of models to be monitored. criterion (Optional[torch.F]): The loss function being optimized (optional). log (Optional[Literal["gradients", "parameters", "all"]]): Specifies whether to log "gradients", "parameters", or "all". Set to None to disable logging. (default="gradients") log_freq (int): Frequency (in batches) to log gradients and parameters. (default=1000) idx (Optional[int]): Index used when tracking multiple models with `wandb.watch`. (default=None) log_graph (bool): Whether to log the model's computational graph. (default=False) |
+|  `models` | A single model or a sequence of models to be monitored. | 
+|`criterion `| The loss function being optimized (optional). |
+|`log `| Specifies whether to log "gradients", "parameters", or "all". Set to None to disable logging. (default="gradients") |
+|`log_freq `|: Frequency (in batches) to log gradients and parameters. (default=1000)| 
```

Comment:
I'm doing a little research about what this option is about. The description is confusing me.